api/handler/acl.go

```diff
@@ -154,16 +154,6 @@ type ServiceRecord struct {
 	GroupRecordsLength int
 }
 
-func (s ServiceRecord) ToEACLRecord() *eacl.Record {
```

Comment:
Do you need `ServiceRecord` type after it?

Reply:
It is still in use [here](https://github.com/nspcc-dev/neofs-s3-gw/blob/bb77e306f43683d3049d7fc8f319d1e9b7249f0a/api/handler/acl.go#L868). On the one hand we should remove all system role, on the other hand - what if some eACL table contains it? Will this code broke something if we remove this usage?

Reply:
Removing this code doesn't break the tests, but I'm not sure, what do you think. @roman-khimov

Reply:
Hmm. S3 is not in production, so we're not likely to get issues missing this code (there no S3-based EACLs that have it). At the same time it's still a part of the protocol and we can keep it (just in case) until it goes out.